templates/page_object.rb

```diff
@@ -0,0 +1,7 @@
+class PageObject
```

Comment:
> I feel like we're arguing, which wasn't my intention

I disagree!

![](https://upload.wikimedia.org/wikipedia/en/8/85/Argument_Clinic.png)

> I have actually copied this exact code into a lot of projects, though. Perhaps this deserves its own Trello card?

We've proven that some people are interested in page objects, so I think it would be worthwhile to try and see what that looks like as a library. Extracting the real code you've already used on several projects seems like a good place to start.

I think a Trello card to track that pilot project would be a good idea as well.